code/numbers.py

```diff
@@ -116,13 +116,29 @@ def number_small(m):
             result += teens_map[word]
     return result
 
-@ctx.capture('number', rule=f'<number_small> [{alt_scales} ([and] (<number_small> | {alt_scales} | <number_small> {alt_scales}))*]')
-def number(m):
+@ctx.capture('self.number_scaled', rule=f'(<digits> | [<digits>] <number_small> [{alt_scales} ([and] (<number_small> | {alt_scales} | <number_small> {alt_scales}))*])')
+def number_scaled(m):
     return fuse_num(fuse_scale(fuse_num(fuse_scale(list(m), 3))))[0]
 
+# This rule offers more colloquial number speaking when combined with a command
+# like: "go to line <number>"
+# Example: " one one five            " == 115
+#          " one fifteen             " == 115
+#          " one hundred and fifteen " == 115
+@ctx.capture('number', rule=f'(<digits> | [<digits>] <user.number_scaled>)$')
```

Comment:
Is the $ necessary here?

Reply:
I removed it. originally i was using that to test what seemed like a time out between saying digits in the actual scaled number, but i think it was due to the new version of town having an overly aggressive time out